VS.Data.Sqlite/SqliteObjectIdentifierConverter.cs

```diff
@@ -20,7 +20,7 @@ protected override string FormatPart(string typeName, object identifierPart, Dat
             if (format.HasFlag(DataObjectIdentifierFormat.WithQuotes)
                 && RequiresQuoting(identifierPartString))
             {
-                // TODO: Why doesn't Query Designer actually use this?
+                // TODO: Do we need this at all now?
```

Comment:
We can try removing it. I think this was just here to prevent DDEX from throwing since the driver doesn't implement DbDataAdapter. But I can't remember.

Reply:
Let's keep it 😃

Reply:
Updated comment